README.rst

```diff
@@ -3,8 +3,33 @@ adb-utils
 
 Utility and Service scripts for the `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_.
 
+adb-utils provides following content to ADB Vagrant box
+
+* OpenShift as a service
+
+  adb-utils uses OpenShift docker containers and provides systemd service unit file which inludes some setup scripts for the purpose. With the systemd unit file, OpenShift can be configured and run in ADB with just "systemctl start openshift" command.
```

Comment:
adb-utils uses a systemd service unit file to launch and setup a containerized version of OpenShift. With the systemd unit file, OpenShift can be configured and run in the ADB by issuing the following command: `systemctl start openshift`